src/Components/Composer.js

```diff
@@ -0,0 +1,90 @@
+// might want to create an edit post option also - need to change from push to update
+
+import { useState } from "react";
+import { database, storage } from "../firebase/firebase";
+import { ref as sRef, uploadBytes, getDownloadURL,} from 'firebase/storage';
+import { push, ref } from "firebase/database";
+
+const DB_FEED_KEY = "feed";
+const DUMMY_USERID = "dummyuser" // to use these as subs
+const DUMMY_PAIRID = 'dummypair' // to use these as subs
+
+//<Composer postContent = {post} />
+export function Composer(props) {
+    const [formInfo, setFormInfo] = useState({
+        file: props.postContent ? props.postContent.val.file : null,
+        postMessage: props.postContent ? props.postContent.val.message : '',
+        date: props.postContent ? props.postContent.val.date : null,
+        tags: props.postContent ? props.postContent.val.tags : '',
+    })
```

Comment:
Don't quite understand this - why are you passing props from FeedPage into Composer? Shouldnt the form data be captured directly by composer?